crowdin/messages.json

```diff
@@ -2185,10 +2167,11 @@
   "-370313455": "The Deriv MT5 Standard account offers new and experienced traders high leverage and variable spreads for maximum flexibility.<br />The Deriv MT5 Advanced account is a 100% A Book account where your trades are passed straight through to the market, giving you direct access to forex liquidity providers.<br />The Deriv MT5 Synthetic Indices account allows you to trade contracts for difference (CFDs) on synthetic indices that mimic real-world movements. It is available for trading 24/7 and audited for fairness by an independent third party.",
   "-1508841947": "DTrader allows you to trade more than 50 assets in the form of digitals, multiplier, and lookback options.<br />Deriv MT5 is a multi-asset trading platform that you can use to trade spot forex and contracts for difference (CFDs) with leverage.",
   "-2073101307": "<0>View fewer questions</0>",
+  "-369631423": "Frequently asked questions - ${untranslate_category}",
```

Comment:
this shouldn't be like this, please use the interpolation feature of i18next, thanks :pray: